File: leetcode-algorithms/628. Maximum Product of Three Numbers/solution.py
def maximumProduct(nums):
	"""
	:type nums: List[int]
	:rtype: int
	"""
	nums.sort()

	return max(nums[-1] * nums[-2] * nums[-3], nums[0] * nums[1] * nums[-1])

    

if __name__ == '__main__':
	nums = [-4,-3,-2,-1,60]
	res = maximumProduct(nums)

	print(res)

File: leetcode-algorithms/628. Maximum Product of Three Numbers/solution.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int maximumProduct(vector<int>& nums) {
	sort(nums.begin(), nums.end());
	int len = nums.size();

	return max(nums[0]*nums[1]*nums[len-1],nums[len-1]*nums[len-2]*nums[len-3]);

}

int main()
{
	vector<int> s{-4,-3,-2,-1,60};

	int res = maximumProduct(s);

	cout<<res<<endl;

	return 0;
}